src/main/java/org/fisco/bcos/sdk/crypto/keypair/CryptoKeyPair.java

```diff
@@ -0,0 +1,101 @@
+/**
+ * Copyright 2014-2020 [fisco-dev]
+ *
+ * <p>Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file
+ * except in compliance with the License. You may obtain a copy of the License at
+ *
+ * <p>http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * <p>Unless required by applicable law or agreed to in writing, software distributed under the
+ * License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.fisco.bcos.sdk.crypto.keypair;
+
+import java.math.BigInteger;
+import java.util.Objects;
+import org.fisco.bcos.sdk.crypto.CryptoNativeResult;
+
+public abstract class CryptoKeyPair {
```

Comment:
Instead to identify a "CryptoKeyPair" by yourself, you can consider use 'java.security.KeyPair'.
The 'org.bouncycastle' package can help you read KeyPair from PEM file and P12 file. If you would like to use this function, maybe you can consider to use 'java.security.KeyPair' directly.

Reply:
Ok. I will fix it in the next PR.